frontend/app/templates/components/forms/signup-form.hbs

```diff
@@ -35,13 +35,15 @@
                 <div class="field required">
                     <div class="ui left icon input">
                         <i class="lock icon"></i>
-                        {{input type="password" id="pwdConfirm" value=password_repeat name="password_repeat" placeholder="Re-enter Password"}}
+                        {{input type="password" id="pwdConfirm" value=password_repeat name="password_repeat" placeholder="Re-enter Password" }}
                         <div class="ui small basic icon buttons">
                             <button class="ui button" id="eye2" {{ action 'showPasswordSignupConfirm'}}>
                                 <i class="eye slash icon"></i>
                             </button>
                         </div>
                     </div>
+                            <div id="msg" style="text-align:center"></div>
```

Comment:
Make proper indentation.

Reply:
@rajvaibhavdubey Please do the changes. :+1: